102-create-expressroute-circuit-private-peering/azuredeploy.json

```diff
@@ -0,0 +1,131 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "circuitName": {
+            "type": "string",
+            "metadata": {
+                "description": "This is the name of the ExpressRoute circuit"
+            }
+        },
+        "serviceProviderName": {
+            "type": "string",
+            "metadata": {
+                "description": "This is the name of the ExpressRoute Service Provider. It must exactly match one of the Service Providers from List ExpressRoute Service Providers API call."
+            }
+        },
+        "peeringLocation": {
+            "type": "string",
+            "metadata": {
+                "description": "This is the name of the peering location and not the ARM resource location. It must exactly match one of the available peering locations from List ExpressRoute Service Providers API call."
+            }
+        },
+        "bandwidthInMbps": {
+            "type": "int",
+            "metadata": {
+                "description": "This is the bandwidth in Mbps of the circuit being created. It must exactly match one of the available bandwidth offers List ExpressRoute Service Providers API call."
+            }
+        },
+        "location": {
+            "type": "string",
+            "defaultValue": "Brazil South",
```

Comment:
Please remove the default location and let the users choose a location from a list of allowed values